superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -188,19 +188,21 @@ class BigNumberVis extends PureComponent<BigNumberVizProps> {
     );
   }
 
-  renderSubtitle(maxHeight: number) {
-    const { subtitle, width } = this.props;
+  rendermetricComparisonSummary(maxHeight: number) {
+    const { subheader, width } = this.props;
     let fontSize = 0;
 
-    if (subtitle) {
+    const text = subheader;
+
+    if (text) {
       const container = this.createTemporaryContainer();
       document.body.append(container);
       try {
         fontSize = computeMaxFontSize({
-          text: subtitle,
+          text,
           maxWidth: width * 0.9,
           maxHeight,
-          className: 'subtitle-line',
+          className: 'subheader-line',
```

Comment:
```suggestion
          className: 'metrics-comparison-line',
```
Can we also rename this?

Reply:
I restored the classname to how it was originally so that it does not break any css customizations that users will have?